.vsts-dotnet-ci.yml

```diff
@@ -0,0 +1,21 @@
+resources:
+  containers:
+  - container: rhel7
+    image: microsoft/dotnet-buildtools-prereqs:centos-7-b46d863-20180719033416
+
+phases:
+- template: /build/ci/phase-template.yml
+  parameters:
+    name: RHEL7
```

Comment:
Let's name this leg the "Linux" leg.  RHEL7 is sort of a lie because we are using CentOS.

Reply:
Ok, sounds good.